src/battle-log.ts

```diff
@@ -720,6 +721,8 @@ class BattleLog {
 			'psicon::type': 0,
 			'psicon::category': 0,
 			'username::name': 0,
+			'youtube::video': 0,
+			'iframe::src': 0,
```

Comment:
Why is this necessary? "`<iframe>` is still banned serverside" is meaningless because side servers can change that code.

Reply:
It's necessary because otherwise the `src` element was removed. I didn't know of another way to handle it, though i asked on Discord :p

Reply:
Your problem is that you're running `sanitizeAttribs`. Look at where `sanitizeAttribs` is executed and you should be able to figure out the correct way to do this.

Reply:
Doesn't Caja do that internally?

Reply:
Oh, no, i'm wrong. I'll sort this out.